### PR TITLE
Added comments to classes

### PR DESCRIPTION
**Pull request by GlebEmelianov**

### CODE REVIEW

File: src/main/java/zoo/Bird.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+public class Bird extends Animal{
+    private String nameOfBird;
+
+    public Bird(String nameOfBird, String action, String move, String breathe) {
+        super(breathe, move, action);
+        this.nameOfBird = nameOfBird;
+    }
+
+    public String getAction() {return action;};
```
Comment (Fulcrum81) [inline src/main/java/zoo/Bird.java]: Unnecessary ";" after a curly brace, please remove

File: src/main/java/zoo/Bird.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+    public String getNameOfFish() {return nameOfBird;};
```
Comment (Fulcrum81) [inline src/main/java/zoo/Bird.java]: Unnecessary ";" after a curly brace, please remove

File: src/main/java/zoo/Fish.java [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+public class Fish extends Animal {
+    private String nameOfFish;
+
+    public Fish(String nameOfFish, String action, String move, String breathe) {
+        super(breathe, move, action);
+        this.nameOfFish = nameOfFish;
+    }
+
+    public String getAction() {return action;};
```
Comment (Fulcrum81) [inline src/main/java/zoo/Fish.java]: Unnecessary ";" after a curly brace, please remove

File: src/main/java/zoo/Fish.java [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+    public String getNameOfFish() {return nameOfFish;};
```
Comment (Fulcrum81) [inline src/main/java/zoo/Fish.java]: Unnecessary ";" after a curly brace, please remove

File: src/main/java/zoo/Main.java [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+    public static void main(String[] args) {
+
+
+        int pointOfMenu = 0;
+
+        String[] listOfAnimals = {"Lion", "Carp", "Parrot"};
+
+        Scanner scanner = new Scanner(System.in);
+
+        Mammal Lion = new Mammal("Lion", "roars", "earth", "air");
```
Comment (Fulcrum81) [inline src/main/java/zoo/Main.java]: Here and below: remember the naming conventions, this is just a variable, the name should be in camelCase

**Review by Fulcrum81 — CHANGES_REQUESTED** [review]
Conceptually: Can I create a Lion in this program that meows, breathes water and flies? I think I can, but I shouldn't be able to do that.
The certain animals should also be implemented as classes so that their actions are predefined and cannot be changed.
If you have need any additional clarification please feel free to ask

**Comment by GlebEmelianov** [comment]
> Conceptually: Can I create a Lion in this program that meows, breathes water and flies? I think I can, but I shouldn't be able to do that. The certain animals should also be implemented as classes so that their actions are predefined and cannot be changed. If you have need any additional clarification please feel free to ask

Added classes for each animal with their unique sound

**Comment by GlebEmelianov** [comment]
Made changes in Zoo code. Added test frameworks and XML files for TestNG

**Pull request merged**